rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -544,11 +544,11 @@ DROP PROCEDURE IF EXISTS \`check_access\`//
 CREATE PROCEDURE \`check_access\`(alias_email VARCHAR(254))
```

Comment:
You may need to update to:

CREATE PROCEDURE \`check_access\`(alias_email VARCHAR(254) charset utf8)